release/release_tests.yaml

```diff
@@ -4448,7 +4476,8 @@
   python: "3.7"
 
   cluster:
-    byod: true
+    byod: 
+      type: cpu
```

Comment:
what does this mean? this is for the cluster type? what does this control?

Reply:
it controls the byod image (either ray-ml:x.x.x-gpu or ray:x.x.x-cpu)